arbor/backends/event_stream_base.hpp

```diff
@@ -20,8 +18,7 @@ template <typename Event>
 struct event_stream_base {
     using size_type = std::size_t;
```

Comment:
This seem redundant or at least underutilized below.

Reply:
good point